packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -480,7 +513,8 @@ void main() {
     await tester.pumpWidget(build(pageSnapping: true));
```

Comment:
I believe we should make sure that pageDidChange does not trigger before turning snapping back on. Can you add an expect for that here?

Reply:
Sorry for my mistakes.Actually it did trigger.It was a bug and I have fixed it in the latest commit.